src/background/controllers/injectionController.ts

```diff
@@ -0,0 +1,69 @@
+import QryptoController from '.';
+import IController from './iController';
+
+/*
+* Content scripts are injected automatically through the manifest.json when a page
+* is loaded, this controller is for handling pages that were loaded prior to
+* installation of the extension.
+*/
+
+export default class InjectionController extends IController {
+  private chromeManifest: any;
+
+  constructor(main: QryptoController) {
+    super('injection', main);
+
+    this.chromeManifest = chrome.runtime.getManifest();
+
+    /* Inject content scripts and add event listeners to all existing tabs.
+    Note that in some cases these pages may already have the content script
+    (i.e. if the extension was uninstalled/reinstalled),
+    so we are doubling/re-injecting the content scripts. This has not caused any
+    problems at present, but a simple alternative would be to refresh all the tabs.
+    */
+    chrome.runtime.onInstalled.addListener((details) => {
+      /*
+      * if extension is being newly installed or prior version was before 1.3.1
+      * (window.qrypto.account was added in v1.3.1)
+      */
+      if (details.reason === 'install'
+        || (details.reason === 'update' && details.previousVersion! < '1.3.1')) {
+        this.injectIntoAllTabs();
```

Comment:
I think a `window.location.reload()` feels like a better solution here. The case I am thinking is that if there is incompatible code by having both the new and old contentscripts injected. Seems like it would be cleaner to just have one contentscript injected. What do you think?